driver/config/config.go

```diff
@@ -826,6 +830,18 @@ func (p *Config) CourierSMTPHeaders() map[string]string {
 	return p.p.StringMap(ViperKeyCourierSMTPHeaders)
 }
 
+func (p *Config) SamlPublicCertPath() *url.URL {
```

Comment:
In Go, abbreviations are always uppercase :)

```suggestion
func (p *Config) SAMLPublicCertPath() *url.URL {
```

Reply:
Ahh thank you I didn't know